### ISSUE TITLE
Icons are blurry and some seem to be the wrong size

### ISSUE THREAD

**Issue by NVLudwig** [unoplatform/uno.todo#164]
## Current behavior

On web the icons are pixelated and blurry some icons don't seem to be the size as others.

## Expected behavior

Clean sharp icons (vectors if possible) all on the same size factor

## How to reproduce it (as minimally and precisely as possible)

Open setting observe the close icon.
Open a list detail Observe the Edit and Delete Task button in comparison to the search icon on the left
![Screen Shot 2022-04-28 at 10 52 32 AM](https://user-images.githubusercontent.com/29584087/165971980-5d22c394-b5c4-441d-9fd5-58fd114c9219.png)
![Screen Shot 2022-04-28 at 10 17 05 AM](https://user-images.githubusercontent.com/29584087/165972121-a29daf7e-5a67-430d-9685-ae2967e54df4.png)

## Environment

MacOS
 Chrome Version 100.0.4896.75 (Official Build) (x86_64) 
 Private window
Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [X] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by NVLudwig** [comment]
@agneszitte-nventive what action is required of me here?

**Comment by kazo0** [comment]
@agneszitte-nventive Correct, the NavigationBar only supports BitmapIcons right now. So we would need to have scaled png assets for those icons

**Comment by NVLudwig** [comment]
I asked Antonio to produce all assets in Dark and Light including NavigationBar icons

**Comment by AntonioBonillaHorus** [comment]
I'm done, let me know if everything is ok. Thank you. (be careful that some icons are white and cannot be seen)
